module.rb

```diff
@@ -0,0 +1,23 @@
+module MyEnumerable
+  def all?
+    each do |e|
+      return false unless yield(e)
+    end
+    true
+  end
+
+  def any?
+    each do |e|
+      return false if yield(e)
+    end
+    true
+  end
+
+  def filter?
```

Comment:
- [ ] This method works fine but the name shouldn't have a `?` at the end, to make it the same as the standard filter method. Kindly fix this. :pray: